.gitignore

```diff
@@ -2,3 +2,4 @@
 /.phpunit.result.cache
 /composer.lock
 /vendor
+.idea/
```

Comment:
this is something that belongs in your global gitignore. Can't expect everyone to use an intelij IDE or we can't add ignores for all OS / IDEs here.

Reply:
@hannesvdvreken Thanks, I didn't realize there was a global .gitignore